preprocess-data.js

```diff
@@ -1,6 +1,13 @@
 const fs = require('fs').promises;
 const fetch = require('node-fetch');
 
+const baseURL = "https://hg.mozilla.org/mozilla-central/";
+
+const filePaths = [
+  "remote/test/puppeteer-expected.json",
+  "remote/puppeteer-expected.json",
+];
+
```

Comment:
style nit for consistency: 

```suggestion
const baseURL = 'https://hg.mozilla.org/mozilla-central/';

const filePaths = [
  'remote/test/puppeteer-expected.json',
  'remote/puppeteer-expected.json',
];

```